cypress/integration/apprenticeFilterSpec.js

```diff
@@ -0,0 +1,75 @@
+/// <reference types="cypress" />
+
+describe('Should filter Apprenticeships correctly', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+  })
+
+  it('filters all', function() {
+    cy.get(`#filters-all`).click();  
+    cy.get('.list__item__content-category').contains('software')
```

Comment:
Nice job! I think we could increase the coverage here by leveraging the `filters` array that you've defined below. If we pull that declaration up to the top, we could do something like:

```
    filters.forEach((filter) => {
      cy.get('.list__item__content-category').contains(filter)
    })
```